Adding basic examples for proxy (#905)
OctopusDeploy/docs: docs/octopus-rest-api/octopus.server.exe-command-line/proxy.md

      --help                 Show detailed help for this command
```

## Basic examples

> This example will enable Octopus Server to use a proxy server using the default Internet Explorer proxy

Reviewer: ```suggestion

This example enables Octopus Server to use a proxy server using the default Internet Explorer proxy:
```

      --help                 Show detailed help for this command
```

## Basic examples

This example enables Octopus Server to use a proxy server using the default Internet Explorer proxy:

```text
octopus.server proxy --proxyEnable="true"
```

> This example will enable Octopus Server to use a defined proxy server for instance MyNewInstance

Reviewer: ```suggestion
This example enables Octopus Server to use a defined proxy server for instance `MyNewInstance`:
```

```
octopus.server proxy --proxyEnable="true" --proxyHost="MyProxyServer" --instance="MyNewInstance"
```

> This example will disable use of a proxy sever on the default instance

Reviewer: ```suggestion
This example prevents a proxy server from being used on the default instance:
```
